src/main/java/uk/gov/pay/directdebit/payments/model/GoCardlessEvent.java

```diff
@@ -12,9 +12,24 @@
     //todo action should be typed (see https://developer.gocardless.com/api-reference/#events-payment-actions and the equivalent for other resource_types
     private String action;
     private GoCardlessResourceType resourceType;
+    private String resourceId;
```

Comment:
I think the `GoCardlessEvent` can be considered immutable, so all fields can be `final` and no need to provide setters.